### PR TITLE
fixed issue where new group members were not pupulated

### PR DESCRIPTION
**Pull request by anatoliyg**

### CODE REVIEW

File: htdocs/css/bootstrap.css [hunk 1]
```diff
@@ -3867,8 +3867,8 @@ textarea.input-group-sm > .input-group-btn > .btn {
 }
 
 .nav-tabs > li.active > a,
-.nav-tabs > li.active > a:hover,
-.nav-tabs > li.active > a:focus {
+.nav-tabs > li.active > a,
+.nav-tabs > li.active > a {
```
Comment (gordonwoodhull) [inline htdocs/css/bootstrap.css]: we avoid changing bootstrap.css, and any other external dependencies, directly - which is the original reason we have bootstrap-rcloud-tweaks.css. I'm moving this rule to bootstrap-rcloud-tweaks.

Comment (gordonwoodhull) [inline htdocs/css/bootstrap.css]: although i see this was not the first change..! reverting those too.

File: htdocs/js/ui/notebook_protection.js [hunk 2]
```diff
@@ -1,88 +1,113 @@
...
-    this.passedData = null;
-    this.appInited = false;
-    this.theApp = null;
-
-
     //set from outside
+    this.defaultCryptogroup = null;
+    this.defaultNotebook = null;
     this.userId;
     this.userLogin;
```
Comment (gordonwoodhull) [inline htdocs/js/ui/notebook_protection.js]: these two lines don't do anything. changing them to initialize to null.

Comment (anatoliyg) [inline htdocs/js/ui/notebook_protection.js]: for sure, all of those types of declarations should be initted with either null, '', [] or {}.

**Pull request merged**